ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Basics.hs

```diff
@@ -54,6 +56,7 @@ import           Ouroboros.Consensus.TypeFamilyWrappers
 import           Ouroboros.Consensus.Util (ShowProxy)
 import           Ouroboros.Consensus.Util.SOP (fn_5)
 
+import           Cardano.Binary
```

Comment:
Would you move this up to the appropriate group of imports? There's a discussion in the https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md

It's preferred but not crucial for new modules, but I'd like to preserve the grouping for those modules that already have it. Thanks.